### ISSUE TITLE
Linter for Processing IDE Python mode?

### ISSUE THREAD

**Issue by scoobers** [jdf/processing.py#381]
Hello. Any chance of getting a linter (i.e., a continuous error checker) for the Processing IDE's Python mode? The IDE's native Java mode has a linter, and so does the p5 mode add-on. Would that be a ton of work to implement?

**Comment by villares** [comment]
Yeah, that would be cool, but I'm afraid it would be a ton of work.

**Comment by jeremydouglass** [comment]
One alternative is to use [p5py](https://github.com/p5py/p5) in any python 3 IDE / programming editor, and then use a python linter / plugin for that editor to do linting (in vscode, atom, sublime, textmate, et cetera).
